<launch>
  <node name="radar_1_publisher" pkg="radar_driver" type="radar_IMEC.py" output="screen"/>
</launch>
